### DISCUSSION TITLE
Mouse buttons

### DISCUSSION THREAD

**Discussion by AnonimePL** [cafali/SnapKey#77] [New Features & Ideas]
Is there any way to do it with LMB and RMB?
I searched on the internet, but I could't find ASCII code for them.
Maybe it needs to make some changes in the code?

**Comment by minteeaa** [comment]
would require extra handling of mouse buttons - they aren't part of the keyboard's keycode layout

not sure if this is built into SOCD or Snap Tap, though - not to "copy" those features, but SnapKey is literally meant to make *parity* of those features on all keyboards, so if it doesn't exist within Razer's or Wooting's implementation then I'm not sure it should be put into SnapKey

ultimately this is up to the maintainer, cafali; I don't really see a real use case for myself for this feature

**Comment by cafali** [comment]
I also don't see it as a necessary feature. Additionally, Snapkey was designed to work with keyboards. The only thing I could suggest is finding software that lets you rebind your mouse keys to your keyboard keys, then making changes in the config file to see if that works for you.
